packages/backend/src/queryBuilder.test.ts

```diff
@@ -409,9 +409,10 @@ describe('with custom dimensions', () => {
     it('getCustomDimensionSql with custom dimension', () => {
         expect(
             getCustomDimensionSql({
+                warehouseClient: bigqueryClientMock,
+
                 explore: EXPLORE,
                 compiledMetricQuery: METRIC_QUERY_WITH_CUSTOM_DIMENSION,
-                fieldQuoteChar: '"',
```

Comment:
Could you explain what this was used for?

Reply:
This is the character the warehouse uses to wrap their fields , eg `"table.column"`